### PR TITLE
Fixed 500 error on fetching QuoteCart for Sweden market.

### PR DESCRIPTION
**Pull request by maksym-krupenko**
# Jira Issue: []

## What?
-

## Why?
-

## Optional checklist
- [ ] Unit tests written
- [ ] Tested locally
- [ ] Codescouted

### CODE REVIEW

File: src/resolvers/quote-cart/quoteCart.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import {
   CreateQuoteBundleResult,
   CreateQuoteCartAccessTokenResult,
   CreateQuoteCartResult,
-  Incentive,
+  Incentive, Provider,
```
Comment (robinandeer) [inline src/resolvers/quote-cart/quoteCart.ts:22]: nit: should be split on two lines

File: src/resolvers/quote-cart/quoteCart.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ import { factory } from '../../utils/log'
 import { getQuoteBundle } from '../quote-bundle/quoteBundle'
 import { ResolverSubset } from '../ResolverSubset'
 import {resolveLocaleFromAcceptLanguage} from "../utils";
+import { AvailablePaymentMethodsResponse } from "../../api/upstreams/paymentService";
```
Comment (robinandeer) [inline src/resolvers/quote-cart/quoteCart.ts:32]: nit: these use double instead of single quotes -- perhaps your prettier setup isn't working?

**Review by robinandeer — APPROVED** [review]
Noticed some syntax quirks but otherwise looks good!

**Review by XSwifter — APPROVED** [review]

File: src/resolvers/quote-cart/quoteCart.ts (hunk 1 above)
Comment (maksym-krupenko) [reply src/resolvers/quote-cart/quoteCart.ts:22]: yeah, my prettier plugin was misconfigured

**Pull request merged**